controllers/session/conditions.go

```diff
@@ -0,0 +1,86 @@
+package session
+
+import (
+	"strconv"
+
+	istiov1alpha1 "github.com/maistra/istio-workspace/api/maistra/v1alpha1"
+	"github.com/maistra/istio-workspace/pkg/model"
+)
+
+func createConditionForLocatedRef(ref model.Ref, located model.LocatorStatus) istiov1alpha1.Condition {
+	message := located.Kind + "/" + located.Name + " status " + ref.KindName.String() + ": "
+	reason := toReason(located.Action)
+	typeStr := string(located.Action)
+	status := "true"
```

Comment:
Should this be false since we haven't done anything yet?

Reply:
My understanding was - it was successfully located. Isn't it what we should report here?

Reply:
Yes and now. The action is "modify" and it's not done yet..

Reply:
If it's "yes and no" we have a bit of an abstraction problem :)

Reply:
Maybe `started`, `pending`, `running` and `finished` should be better than `true|false` status in such a case?

Reply:
The overall status.state is Processing, Success, Failed. What we're talking about here is a condition; Is it done; bool

We could change the reason a bit to indicate something different from when it's processed. Like 'Ready to start'

Reply:
As discussed - reworded `reason` in 917ec03